app/assets/stylesheets/views/_simple-smart-answer.scss

```diff
@@ -0,0 +1,8 @@
+.step {
+  .next-question {
+    button {
+      font-size: 19px;
+      line-height: 1.3684210526;
```

Comment:
This is a weird number - where we see this in a browser it's usually a result of something being given as a % of something else, it doesn't look like something we should be hard coding.

Reply:
Agreed, the previous button seemed to use `line-height: 1.25` which I think we can use to have the same effect.

I'd also ask whether we need these styles here - it'd be better not to be overriding component styles for specific formats.

Reply:
I've amended the line height to `1.25` here https://github.com/alphagov/frontend/pull/1430/commits/0f129bbc6b4f76f2e95babacf347baad3e721ad2
The button font size was controlled by the `big` css class, we don't make provision for passing additional css classes to the button component so either an override is necessary or we accept the stylistic differences.